src/main/webapp/find-events.html

```diff
@@ -64,54 +64,57 @@ <h1 id="find-events-title" tabindex="0">find events</h1>
           </div>
         </div>
         <div id="all-posts">
-          <button id="create-post-button" type="button" tabindex="0" aria-label="create a post">create a post</button>
+          <button aria-label="create a post" id="create-post-button" tabindex="0" type="button">create a post</button>
         </div>
       </div>
     </div>
 
     <!-- modal -->
     <div id="modal-background">
-      <div id="modal-create-post" tabindex="0" aria-label="create a post modal">
-        <button id="modal-close" type="button" tabindex="0" aria-label="close modal">&#10005;</button>
+      <div aria-label="create a post modal" id="modal-create-post" tabindex="0">
+        <button aria-label="close modal" id="modal-close" tabindex="0" type="button">&#10005;</button>
         <h1 id="modal-title" tabindex="0">Create a Post</h1>
         <form id="modal-form" method="post">
-          <input type="text" id="modal-org-name" name="organizationName" placeholder="organization name" tabindex="0" aria-label="organization name" required><br>
+          <input aria-label="organization name" id="modal-org-name" maxlength="100" name="organizationName" placeholder="organization name" tabindex="0" type="text" required><br>
```

Comment:
Should organization name length be max 100? :0

Reply:
100 characters is the amount of characters in this comment! I agree, I'll limit it to 50 characters.

Reply:
Fifty characters is this many words which is fine.

Reply:
So if the organizations have super long acronyms as names they can list it.

Reply:
^ that was 75 characters 😄